content/advent-2015/generate-free-tls-certs-with-lego.md

```diff
@@ -85,15 +85,13 @@ The email address is optional, but highly recommended so you can recover your ac
 Now we can create a client for the user that will speak ACME with the CA server.
 
 ```go
-client, err := acme.NewClient("https://acme-v01.api.letsencrypt.org/directory", &myUser, rsaKeySize, "")
+client, err := acme.NewClient("https://acme-v01.api.letsencrypt.org/directory", &myUser, rsaKeySize)
 if err != nil {
     log.Fatal(err)
 }
 ```
 
-The acme package expects the URL to the CA server's ACME directory. In this case, we're using Let's Encrypt's live endpoint. If you're just testing, use their staging endpoint, `https://acme-staging.api.letsencrypt.org/directory`. The staging endpoint is mostly the same, except it returns untrusted certificates and doesn't have the same rate limits.
-
-We also pass in our user and the size of the key for generated certs. The last argument to NewClient is an optional alternate port to bind to; we left it blank to use the default port, which depends on the challenge (see below). You can leave it blank if your Go program has permission to bind to low ports.
+The acme package expects the URL to the CA server's ACME directory. In this case, we're using Let's Encrypt's live endpoint. If you're just testing, use their staging endpoint, `https://acme-staging.api.letsencrypt.org/directory`. The staging endpoint is mostly the same, except it returns untrusted certificates and doesn't have the same rate limits. We also pass in our user and the size of the key for generated certs.
```

Comment:
Yeah, you're right. I was just explaining the arguments in this paragraph; the first one requires more explanation.